robottelo/decorators/host.py

```diff
@@ -1,56 +1,35 @@
+"""Implements decorator regarding satellite host"""
 import logging
 import re
-import six
 import unittest2
 
+from backports.functools_lru_cache import lru_cache
```

Comment:
If we are going to rely on external dependency maybe functools32 or [repoze.lru](http://docs.repoze.org/lru/narr.html#decorating-an-expensive-function-call) are the known lru_cache libraries for python2.x